src/Components/Steps/Steps.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+import { observer } from "mobx-react";
+import styles from "./Steps.module.css";
+import type { MediaPresenter } from "../../MediaPresenter";
+import type { OutputPresenter } from "../../OutputPresenter";
+import { showError } from "../ErrorToast/ErrorToast";
+
+type Props = {
+  mediaPresenter: MediaPresenter;
+  outputPresenter: OutputPresenter;
+  openPlayerModal: () => void;
+  closePlayerModal: () => void;
+};
+
+export const Steps: React.FC<Props> = observer(
+  ({ mediaPresenter, outputPresenter, openPlayerModal, closePlayerModal }) => {
+    const handleMediaUpload = async (
+      event: React.ChangeEvent<HTMLInputElement>
+    ) => {
+      if (event.target.files === null) return;
+      const files: FileList = event.target.files;
+      Array.from(files).forEach((file: File) => {
+        if (!mediaPresenter.addFile(file)) {
+          showError(file.name + "is not an acceptable file format");
+        }
+      });
+    };
+
+    const [stepOneOpen, setStepOneOpen] = useState<boolean>(true);
+    const [stepTwoOpen, setStepTwoOpen] = useState<boolean>(false);
+
+    return (
+      <div className={styles.stepsContainer}>
+        <div
+          className={styles.steppingContainer}
+          onClick={() => {
+            if (mediaPresenter.mediaReady) {
+              if(!stepOneOpen) {
+                setStepOneOpen(true);
+              }
+              closePlayerModal();
+            }         
+          }}
+        >
+          <span className={styles.stepNumber}>1</span>
+
+          <p className={styles.stepText}>Add your files</p>
+          <span onClick={()=> {
+            setStepOneOpen(!stepOneOpen);
+          }}>
+            <i className={stepOneOpen ? styles.downArrow : styles.upArrow}></i>
+          </span>
+        </div>
+
+        {stepOneOpen && (
```

Comment:
The current setup seems to allow both step one and two to be open at the same time. How about instead of storing each modals `open` value, you store a `currentStep` state that can have the values `'one' | 'two' | 'none'`?